tests/org.eclipse.n4js.ide.tests/src/org/eclipse/n4js/ide/tests/bugreports/GH_1956_RelinkTModuleLeavingResourceInInvalidState.xtend

```diff
@@ -82,8 +83,16 @@ clearOutput();
 	}
 
 	/** Line/column is zero-based. */
-	def private TextDocumentPositionParams textDocPos(String moduleName, int line, int column) {
-		return new TextDocumentPositionParams(
+	def private HoverParams textDocPosHoverParams(String moduleName, int line, int column) {
+		return new HoverParams(
+			new TextDocumentIdentifier(getFileURIFromModuleName(moduleName).toString),
+			new Position(line, column)
+		);
+	}
+
+	/** Line/column is zero-based. */
```

Comment:
I have 5 warnings about deprecation in the workspace in files `JSONHierarchicalSymbolServiceTest.xtend` and `GH_1956_RelinkTModuleLeavingResourceInInvalidState.xtend`.

Since this is Xtend code, there is no suppress-warnings feature, but would it be possible to avoid these warnings in a different way? Maybe by using a Java utility file that forwards the deprecated methods or something?